teeracle/Cargo.toml

```diff
@@ -124,20 +124,22 @@ path = "../primitives/teeracle"
 default = ['std']
 std = [
     'codec/std',
-    'scale-info/std',
+    'log/std',
     'frame-support/std',
     'frame-system/std',
+    "pallet-teerex/std",
+    'scale-info/std',
     'sp-core/std',
     'sp-io/std',
     'sp-runtime/std',
     'sp-std/std',
     'substrate-fixed/std',
+    "teeracle-primitives/std",
+    "test-utils/std",
```

Comment:
Is test-utils used by the benchmark tests? otherwise we can change this dep to be a dev-dep without opting out of the default-features.

Reply:
Same for the # benchmarking stuff below?

Reply:
No, it is the other way around, benchmarking stuff needs to be a real dependency.

Reply:
test-utils used by the benchmark.

Reply:
Shall I set it under runtime-benchmarks and make it optional?